galaxy.yml

```diff
@@ -19,6 +19,7 @@ tags:
 dependencies:
   amazon.aws: '>=5.1.0'
   community.aws: '>=5.0.0'
+  community.crypto: '>=2.15.1'
```

Comment:
This is not required by the collection, it is needed for the integration tests only and this has already been specified here https://github.com/redhat-cop/cloud.aws_ops/blob/e7e6dd1b9ae257714aedcdc577e31044f937c84e/tests/integration/requirements.yml#L9
```suggestion
```